policyengine_us/variables/gov/states/ky/tax/income/ky_filing_status.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class KyFilingStatus(Enum):
+    SINGLE = "Single"
+    JOINT = "Joint"
+    SEPARATE = "Separate"
+
+
+class ky_filing_status(Variable):
+    value_type = Enum
+    entity = TaxUnit
+    possible_values = KyFilingStatus
+    default_value = KyFilingStatus.SINGLE
+    definition_period = YEAR
+    label = "Filing status for the tax unit in Kentucky"
+    defined_for = StateCode.KY
+
+    def formula(tax_unit, period, parameters):
+        has_spouse = add(tax_unit, period, ["is_tax_unit_spouse"]) > 0
```

Comment:
depending on how the legal code structures it, we may want to reference federal filing_status instead

Reply:
not sure if the legal code structures in a way that we should change it, added the reference

Reply:
not sure that's the best legal code section for it, but added a tax form reference too so i think we're good